PhysicsTools/NanoAOD/python/jetsPAIReD.py

```diff
@@ -0,0 +1,10 @@
+import FWCore.ParameterSet.Config as cms
+from  PhysicsTools.NanoAOD.common_cff import *
+
+pairedJetTable = cms.EDProducer("PAIReDONNXJetTagsProducer")
+
+pairedJetTableTask = cms.Task(pairedJetTable)
+
+pairedJetTableMC = cms.EDProducer("PAIReDONNXJetTagsProducer")
```

Comment:
if there is no specific parameter difference between data and MC, you should keep the same module `pairedJetTable`